### PR TITLE
Save payload

### PR DESCRIPTION
**Pull request by leniatgh**

### CODE REVIEW

**Review by jordan13game — APPROVED** [review]

File: lib/apiScenario/gen/staticApiScenarioGenerator.ts [hunk 1]
```diff
@@ -52,13 +38,13 @@ export class StaticApiScenarioGenerator {
       eraseDescription: false,
       filerTopLevelResourceType: true,
       noExternalDependencyResourceType: true,
-      rules: ["resource-put-delete"],
```
Comment (raych1) [inline lib/apiScenario/gen/staticApiScenarioGenerator.ts:55]: Is this rule used currently? Is it safe to delete? @ruowan

**Comment by raych1** [comment]
@ruowan , there're a bunch of deletions, can you review those changes?

File: lib/apiScenario/gen/staticApiScenarioGenerator.ts (hunk 1 above)
Comment (ruowan) [reply lib/apiScenario/gen/staticApiScenarioGenerator.ts:55]: Currently, we don't use this. Unified pipeline use fixed version of oav. It's safe to delete it.

**Comment by ruowan** [comment]
LGTM. For dependency we rely on restler and report has been already refactor as readme or html format.

**Review by ruowan — APPROVED** [review]

**Pull request merged**